### PR TITLE
Updated docs with supported operating systems.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: docs/pages/faq.mdx [hunk 1]
```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./setup/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport runs on Linux, macOS, and Windows (client only).
+
+|Operating System| Version       |
+|----------------|---------------|
+| macOS          | X             |
```
Comment (russjones) [inline docs/pages/faq.mdx:132]: @codingllama What's the minimum version of macOS `tsh` now supports?

File: docs/pages/faq.mdx [hunk 2]
```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
...
+| Windows        | Y             |
```
Comment (russjones) [inline docs/pages/faq.mdx:133]: @timothyb89 What's the minimum version of Windows needed by `tsh.exe`?

File: docs/pages/faq.mdx (hunk 1 above)
Comment (codingllama) [reply docs/pages/faq.mdx:132]: macOS 10.13, aka High Sierra.

Source: https://github.com/gravitational/teleport/blob/80baab8ce584bb903ccec566d037a816770d6f81/lib/auth/touchid/api_darwin.go#L20

File: docs/pages/faq.mdx [hunk 3]
```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./setup/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport runs on Linux, macOS, and Windows (client only).
```
Comment (ptgott) [inline docs/pages/faq.mdx:128]: Does "client only" also apply to remote use of `tctl`?

**Review by ptgott — APPROVED** [review]
Approved with one question, assuming the placeholders are filled in.

File: docs/pages/faq.mdx (hunk 3 above)
Comment (russjones) [reply docs/pages/faq.mdx]: No, we only support `tsh.exe` on Windows for now. Reworded it a bit, what do you think?

File: docs/pages/faq.mdx (hunk 2 above)
Comment (timothyb89) [reply docs/pages/faq.mdx]: Windows 10, revision 1607 (the 2016 Anniversary update) and later.

**Review by codingllama — APPROVED** [review]

File: docs/pages/faq.mdx [hunk 4]
```diff
@@ -127,13 +127,13 @@ information.
 
 Teleport binaries runs on Linux, macOS, and Windows.
```
Comment (ptgott) [inline docs/pages/faq.mdx:128]:
```suggestion
Teleport binaries run on Linux, macOS, and Windows.
```
Tiny grammar tweak

**Review by zmb3 — APPROVED** [review]

**Pull request merged**